src/model/application.rs

```diff
@@ -11,16 +13,18 @@ pub struct Configuration {
 	pub message_responses: Vec<MessageResponse>,
 }
 
+const CONFIG_PATH: &str = "configuration.json";
```

Comment:
Instead of this, first check in the current directory for the config file, if it does not exist, create a configuration file in the home directory ~/.config/...

Reply:
> Instead of this, first check in the current directory for the config file, if it does not exist, create a configuration file in the home directory ~/.config/...

![image](https://user-images.githubusercontent.com/87065584/178108928-3eba7063-01b7-4e77-a24b-72efd74e5a71.png)

I did something like this